libr/bin/p/bin_lua.c

```diff
@@ -144,7 +144,10 @@ static void addString(const ut8 *buf, ut64 offset, ut64 length, ParseStruct *par
 		return;
 	}
 
-	binstring->string = r_str_newlen ((char *) buf + offset, length);
+	binstring->string = NULL;
```

Comment:
assign this once. use ternary operator line `binstring->string = (length > 0)?  ...: NULL;`